crypto/perlasm/x86_64-xlate.pl

```diff
@@ -289,11 +300,11 @@
 	    $self->{label} =~ s/(?<![\w\$\.])0x([0-9a-f]+)/0$1h/ig;
 	    $self->{label} = "($self->{label})" if ($self->{label} =~ /[\*\+\-\/]/);
 
-	    ($self->{asterisk})					&& ($sz="q") ||
-	    (opcode->mnemonic() =~ /^v?mov([qd])$/)		&& ($sz=$1)  ||
-	    (opcode->mnemonic() =~ /^v?pinsr([qdwb])$/)		&& ($sz=$1)  ||
-	    (opcode->mnemonic() =~ /^vpbroadcast([qdwb])$/)	&& ($sz=$1)  ||
-	    (opcode->mnemonic() =~ /^v(?!perm)[a-z]+[fi]128$/)	&& ($sz="x");
+	    ($self->{asterisk})						&& ($sz="q") ||
+	    ($self->{opcode}->mnemonic() =~ /^v?mov([qd])$/)		&& ($sz=$1)  ||
+	    ($self->{opcode}->mnemonic() =~ /^v?pinsr([qdwb])$/)	&& ($sz=$1)  ||
+	    ($self->{opcode}->mnemonic() =~ /^vpbroadcast([qdwb])$/)	&& ($sz=$1)  ||
+	    ($self->{opcode}->mnemonic() =~ /^v(?!perm)[a-z]+[fi]128$/)	&& ($sz="x");
```

Comment:
I would suggest to `my $mnemonic = $self->{opcode}->mnemonic()` prior this block and then use `$mnemonic` here. This is make text *narrower*.